authors/apps/articles/urls.py

```diff
@@ -5,7 +5,7 @@
 app_name = 'articles'
 
 urlpatterns = [
-    path('articles/', ArticleAPIView.as_view(), name='create'),
-    path('articles/<str:pk>', ArticleDetailsView.as_view(),
+    path('', ArticleAPIView.as_view(), name='create'),
```

Comment:
@pngg Updating these would be going out of scope of your task. Create a chore for this separately on PT

Reply:
Noted